vendor/lib/flag_exists.cmd

```diff
@@ -0,0 +1,131 @@
+@echo off
+setlocal
+
+if "%~1" equ "" goto :wrongSyntax
+
+if not defined CMDER_USER_FLAGS (
+  :: in case nothing was passed to %CMDER_USER_FLAGS%
+  set "CMDER_USER_FLAGS= "
+)
+
+set "feNot=false"
+goto :parseArgument
+
+:doShift
+  shift
```

Comment:
I'm curious, why do you call `shift` in a dedicated subroutine `doShift`?

Reply:
https://github.com/cmderdev/cmder/blob/0d2e16c6536495c9ebc2466b3a15582a9aee45e3/vendor/bin/alias.cmd#L17
https://github.com/cmderdev/cmder/pull/1758/files/39f47870a339c99e7526920ec3e06b875db8890b#diff-6584be369e2738104e1069789ea6bfe4R29

For the detection of `NOT`, copied from your `alias.cmd`.